src/components/Card/Post/PostListFilter.jsx

```diff
@@ -0,0 +1,170 @@
+import { useState } from "react";
+
+import { ERROR_MESSAGE } from "../../../config/constants";
+import KeywordChip from "../../Chip/KeywordChip";
+import Button from "../../UI/Button";
+import Label from "../../UI/Label";
+import PropTypes from "prop-types";
+
+const PostListFilter = ({ filterList, setFilterList }) => {
+  const [tempFilterList, setTempFilterList] = useState({
+    includedKeyword: filterList.includedKeyword,
+    excludedKeyword: filterList.excludedKeyword,
+  });
+  const [inputValue, setInputValue] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+  const [errorMessage, setErrorMessage] = useState({
+    includedKeyword: "",
+    excludedKeyword: "",
+  });
+
+  const handleFilterInputChange = (e, filterType) => {
+    setInputValue((prev) => ({ ...prev, [filterType]: e.target.value }));
+    return;
+  };
+
+  const handleCreateTempFilterSubmit = (e, filterType) => {
+    e.preventDefault();
+
+    if (inputValue[filterType]?.trim() === "") {
+      return;
+    }
```

Comment:
```suggestion
  const trimmedInputValue = inputValue[filterType].trim();

    if (trimmedInputValue  === "") {
      return;
    }
```

위와 같이 trim한 값을 선언 및 초기화 해두고 계속 사용해도 좋을 것 같습니다~

Reply:
다음 코드에서도 활용되므로 `trim()` 처리후 재사용하면 좋겠네요. 감사합니다 :)